teuthology/task/ceph_ansible.py

```diff
@@ -20,12 +20,25 @@ class CephAnsible(Task):
     name = 'ceph_ansible'
 
     _default_playbook = [
+        # run gather facts at the top as required by mgr-role and eventually
+        # stop using _default_playbook and instead use site.yml.sample
+        # from ceph-ansible.git
```

Comment:
I'm confused about the above: we're modifying `_default_playbook` to gather facts first... but what about `site.yml.sample`?